README.md

```diff
@@ -16,9 +16,11 @@ npm install --save @chanzuckerberg/eds
 yarn add @chanzuckerberg/eds
 ```
 
-Import the CSS tokens somewhere in your app, e.g. an `init.ts` or `app.ts` file:
+Import the CSS stylesheet and tokens somewhere in your app, e.g. an `init.ts` or `app.ts` file:
 
 ```js
+import '@chanzuckerberg/eds/lib/index.css';
```

Comment:
One downside of this is you have to import ALL the css, even if you only use one component.

I suspect that's not a big deal for our use case, but need to do some testing to see if unused styles get removed.

Reply:
Coming back to this, I think it'll be the same result in Remix apps regardless of whether we're shipping css modules or bundled css.